session/src/test/java/org/apache/iotdb/session/pool/SessionPoolTest.java

```diff
@@ -164,7 +164,7 @@ private void correctQuery(SessionPool pool) {
 
   @Test
   public void tryIfTheServerIsRestart() {
-    SessionPool pool = new SessionPool("127.0.0.1", 6667, "root", "root", 3, 1, 6000, false, null);
+    SessionPool pool = new SessionPool("127.0.0.1", 6667, "root", "root", 3, 1, 6000, false, null, false);
```

Comment:
Is there some UT for set the enableCache as `true`?

Reply:
I will add UT for this laterly